docs/components/modeler/element-templates/element-template-with-dependencies.md

```diff
@@ -0,0 +1,35 @@
+---
+id: element-templates-with-dependencies
+title: Element templates with dependencies
+description: "Learn what you need to take into account when handling template dependencies."
+---
+
+When creating element templates, you may want to link to a particular resource like a [form](/components/modeler/forms/camunda-forms-reference.md), or pre-populate a [secret](/components/connectors/use-connectors/index.md#using-secrets) expression. Your template might require a particular [job worker](/components/concepts/job-workers.md) to execute an action. These are all different types of dependencies.
+
+Element templates can depend on:
+
+- Resources:
+  - [Camunda forms](/components/modeler/forms/camunda-forms-reference.md): used in user tasks.
+  - [RPA scripts](/components/rpa/overview.md): Used in service tasks.
+  - [BPMN tasks](/components/modeler/bpmn/bpmn.md): used in call activities.
```

Comment:
🔧 While this is somehow technically true, because of call activities, it's only a transitive dependency. I wouldn't cover it here.